src/server/query.c

```diff
@@ -65,17 +65,71 @@ tds_lastpacket(TDSSOCKET * tds)
 	return tds->in_buf[1] != 0;
 }
 
+
+/**
+ * get query packet of a given type
+ * \tds
+ * \param head         extra information to put in a TDS7 header
+ */
+static TDSRET
+tds_get_query_head(TDSSOCKET * tds, TDSHEADERS * head)
+{
+	int more;
+
+	if (IS_TDS72_PLUS(tds->conn)) {
```

Comment:
maybe return here to reduce function indentation?

Reply:
Sure can do, was just following existing code from the query side. I prefer exit-early. but also wanted to follow existing code format.

https://github.com/FreeTDS/freetds/blob/master/src/tds/query.c # tds_start_query_head